Gems/Atom/RHI/Code/Source/RHI/ShaderResourceGroupPool.cpp

```diff
@@ -259,6 +257,8 @@ namespace AZ
             for (uint32_t i = interval.m_min; i < interval.m_max; ++i)
             {
                 ShaderResourceGroup* group = m_groupsToCompile[i];
+                AZ_PROFILE_SCOPE(RHI, "CompileGroupsForInterval %s", group->GetName().GetCStr());
```

Comment:
will that not be too noisy?

Reply:
It is a little noisy, but it was required to find out which SRG compile group was taking 1.5 - 3 ms / frame.